Sushant203/GitPractice
# feat: Create BookNow component for booking process

## Description:
This PR introduces the `BookNow` component which facilitates booking process. Users can view contact information, email and phone details, address.

## Changes:
- Created `BookNow` component in `src/components/BookNow.jsx`.
- Styled the component using Tailwind CSS.
- Added contact details for email and phone, allowing users to initiate communication.

## How to Test:
- Navigate to the `BookNow` component in the app to check the layout and functionality.
